mxtools/handlers.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import pathlib
+
+import dask.array as da
+import h5py
+from area_detector_handlers import HandlerBase
+
+logger = logging.getLogger(__name__)
+
+
+class EigerHandlerMX(HandlerBase):
+    spec = "AD_EIGER_MX"
+
+    def __init__(self, fpath, seq_id):
+        self._seq_id = seq_id
+        # From https://github.com/bluesky/area-detector-handlers/blob/0f47155b31a6b4bf92c1c2b6fe98b5f141194c78/area_detector_handlers/eiger.py#L84  # noqa
+        #
+        #         master_path = Path(f'{self._file_prefix}_{seq_id}_master.h5').absolute()
+        #
+        self._fpath = pathlib.Path(f"{fpath}_{seq_id}_master.h5").absolute()
+        if not self._fpath.is_file():
+            raise RuntimeError(f"File {self._fpath} does not exist")
+
+        # print(f"Eiger master file: {self._fpath}")
```

Comment:
pretty trivial, but also such debug code?

Reply:
We need a proper logger, and this message can go to the debug stream.